src/components/ControlGroup/EnumControl.tsx

```diff
@@ -0,0 +1,24 @@
+type Props = {
+	options: Array<string>;
+	description?: string;
+	setPosition: (position: string) => void;
+};
+
+const EnumControl = ({ description, options, setPosition }: Props) => {
```

Comment:
`export const EnumControl` please and remove line on end want every export in every file to look identical so that the codebase reads as if it's written by 1 person